storage/connectors/bigquery/src/main/java/feast/storage/connectors/bigquery/writer/FeatureSetSpecToTableSchema.java

```diff
@@ -99,7 +102,7 @@ public void processElement(
       createTable(specKey, schema);
```

Comment:
This comment probably comes late, but since table updates are purely additive (and backward compatible), is there a downside to running `createTable` even if the table already exists?

Reply:
I suppose it's no point to run createTable if table already exists. I guess you meant updateTable, but we're already doing it as part of batch load process, which 100% guaranteed to be consistent approach and doing it here - I'm not sure about consistency.

Reply:
Yeah, i meant updateTable. By sketchy consistency do you mean the race condition between in-progress writes and feature-set updates?